workflow/controller/operator.go

```diff
@@ -1437,9 +1472,15 @@ func (woc *wfOperationCtx) addChildNode(parent string, child string) {
 
 // executeResource is runs a kubectl command against a manifest
 func (woc *wfOperationCtx) executeResource(nodeName string, tmpl *wfv1.Template, boundaryID string) *wfv1.NodeStatus {
+	var skipNodeInitialization bool
 	node := woc.getNodeByName(nodeName)
 	if node != nil {
-		return node
+		if strings.Contains(node.Message, exceededQuota) {
```

Comment:
Move the strings.Contains call into a function called ExceededQuota or something?